inc/integrations/class-integrations-manager.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * WP Irving Integrations Manager
+ *
+ * @package WP_Irving
+ */
+
+namespace WP_Irving;
+
+/**
+ * Class for managing integrations in Irving.
+ */
+class Integrations_Manager {
+
+	/**
+	 * Class instance.
+	 *
+	 * @var null|self
+	 */
+	protected static $instance;
+
+	/**
+	 * Get class instance.
+	 *
+	 * @return self
+	 */
+	public static function instance() {
+		if ( ! isset( static::$instance ) ) {
+			static::$instance = new static();
+			static::$instance->setup();
+		}
+		return static::$instance;
+	}
+
+	/**
+	 * Class constructor.
+	 */
+	public function setup() {
+		// Register admin page.
+		add_action( 'admin_menu', [ $this, 'register_admin' ] );
+		// Register settings fields for integrations.
+		add_action( 'admin_init', [ $this, 'register_settings_fields' ], 10, 2 );
```

Comment:
`register_settings_fields()` doesn't accept any params, so you can omit the last two arguments here.

```suggestion
		add_action( 'admin_init', [ $this, 'register_settings_fields' ] );
```